src/features/dataset/state/datasetActions.ts

```diff
@@ -27,6 +33,19 @@ export function downloadLinkFromQriRef(ref: QriRef, body: boolean = false): stri
   return `${API_BASE_URL}/ds/get/${ref.username}/${ref.name}${pathSegment}?format=zip`
 }
 
+function fetchDatasetVersionInfo(ref: QriRef) {
```

Comment:
As per #312, we should call this function `fetchHeader` & we need to add a TODO here explaining that the `collection/get` endpoint is only temporarily returning `VersionInfos`:

```
// fetchHeader fetches a specified `VersionInfo` from the collection
// TODO(boandriy): this endpoint ('collection/get') returns a `VersionInfo`, but
// this return type may change in the future when the specific "dataset details"
// that a user wants to see on the dataset page settles. We may then have a new
// data structure that returns from the 'collection/get' endpoint
```

Again, because this endpoint will not always return `VersionInfo`, it would be inaccurate to call it `fetchDatasetVersionInfo`.